wsmaster/che-core-api-system/src/main/java/org/eclipse/che/api/system/server/ServiceTermination.java

```diff
@@ -20,7 +20,7 @@
  *
  * @author Yevhenii Voevodin
  */
-interface ServiceTermination {
+public interface ServiceTermination {
```

Comment:
Strategy for service termination changed and now certain service could provide the implementation of its termination.